core/include/mm/sp_mem.h

```diff
@@ -0,0 +1,59 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021, Arm Limited.
+ */
+
+#ifndef __MM_SP_MEM_H
+#define __MM_SP_MEM_H
+
+#include <ffa.h>
+#include <mm/fobj.h>
+#include <sys/queue.h>
+
+struct sp_mem_receiver;
+struct sp_mem;
+
+struct sp_mem_receiver {
+	struct ffa_mem_access_perm perm;
+	uint8_t ref_count;
```

Comment:
What kind of refcount is this?

Reply:
It is for keeping track of the amount of times the share has been retrieved. Commented it